### DISCUSSION TITLE
Integrating Mosaico with .NET 6 on IIS

### DISCUSSION THREAD

**Discussion by Chris5gd** [voidlabs/mosaico#668] [Q&A]
Hi, I'm just embarking on a project to integrate Mosaico with a .NET 6 web-based proprietary CRM application.  We use Visual Studio 2022 as IDE.

I'm aware that on Github there are [ASP.NET Core 2](https://github.com/gordon-matt/Mosaico.NetCore) and [ASP.NET MVC5](https://github.com/gordon-matt/Mosaico.Mvc5) sample projects, but we've rolled our own JavaScript event handler, which calls server-side code via an Ajax POST, and I'm struggling to see how to map these samples to our framework.

So I'm looking for some suitable documentation and/or simple samples that don't use out-of-support frameworks or MVC.

The design goal is to use Mosaico to create, edit and save templates to a back end SQL server database, where the back end web app will use the created content to send SMTP emails to mailing list subscribers.  All the back end mailing code is well established - the only issue is content creation.

Any help will be much appreciated!

**Comment by davide-caffaratti** [comment]
I made by php backend and think Is not very complicate, change the php script for .NET

https://github.com/davide-caffaratti/mosaico-php-backend
